perfzero/lib/perfzero/perfzero_config.py

```diff
@@ -70,6 +70,12 @@ def add_setup_parser_arguments(parser):
       help='''Additional specifications to pass to `pip install`. (e.g. pinning certain dependencies)
       Specifications should be semicolon separated: e.g. `numpy==1.16.4;scipy==1.3.1`
       ''')
+  parser.add_argument(
+      '--enable_docker_setup',
```

Comment:
What if you just pass an empty docker path and that indicates you do not want to build docker. 

We could make a change to have it not build the docker by default by having the default path set to none, but I think your usage is a bit different from the external world and passing an empty value seems like it would not be too painful for setup. Let me know.